templates/examples/constellation/text-processing/src/index.ts

```diff
@@ -0,0 +1,146 @@
+import _ from "lodash";
+
+import { ortToConsn } from "./transformers-js/tensor_utils";
+import { structureArray } from "./utils";
+import { pipeline, Pipeline } from "./transformers-js/pipelines";
+
+export interface Env {
+	CONSN: any;
+}
+
+globalThis.binding = {};
+
+var SENTIMENT_ANALYSIS_PIPELINE: Pipeline;
```

Comment:
Its generally best practice to use `let` for reassignable variables, it is block scoped and can help prevent odd bugs from `var`'s hoisting behavior. This in regards to all the use of `var` in the template, just picking a place to mention it.